### ISSUE TITLE
Issues with Imaginary

### ISSUE THREAD

**Issue by luigir-it** [matiasdelellis/facerecognition#692]
I'm running `sudo -u www-data php /var/www/nextcloud/occ face:background_job -u administrator` to start analyzing pictures. But this shows up

`1/8 - Executing task CheckRequirementsTask (Check all requirements)
	An Imaginary service (http://127.0.0.1:9000) was configured to manage temporary images, but it is inaccessible.Check out the service, or set the 'preview_imaginary_url' key appropriately.
Task CheckRequirementsTask signalled we should not continue, bailing out`

running netstat shows 
`tcp        0      0 127.0.0.1:9000          0.0.0.0:*               LISTEN     `

and docker container ls
`987c256e8685   nextcloud/aio-imaginary          "imaginary -return-s…"   5 weeks ago   Up 2 minutes (healthy)   127.0.0.1:9000->9000/tcp                                                 imaginary
`

**Comment by matiasdelellis** [comment]
But can you access to them?
```
[matias@nube ~]$ curl  127.0.0.1:9000
{"imaginary":"dev","bimg":"1.1.7","libvips":"8.14.2"}
[matias@nube ~]$ 
```
Some reported that you also need to configure `allow_local_remote_servers' => true,` on config.php

**Comment by luigir-it** [comment]
> allow_local_remote_servers' => true,

Yes, this was the missing piece. Is it worth to add it to the documentation?

**Comment by mrbrdo** [comment]
@matiasdelellis I had this issue as well and adding that config fixed it. Should definitely be added to docs somewhere (or better yet, improve the error message itself).
